src/qml/components/blockclockdial.cpp

```diff
@@ -15,7 +17,39 @@ BlockClockDial::BlockClockDial(QQuickItem *parent)
 , m_background_color{QColor("#2D2D2D")}
 , m_confirmation_colors{QList<QColor>{}}
 , m_time_tick_color{QColor("#000000")}
+, m_animation_timer{QTimer(this)}
+, m_delay_timer{QTimer(this)}
 {
+    m_animation_timer.setTimerType(Qt::PreciseTimer);
+    m_animation_timer.setInterval(16);
+    m_delay_timer.setTimerType(Qt::PreciseTimer);
+    m_delay_timer.setSingleShot(true);
+    m_delay_timer.setInterval(5000);
+    connect(&m_animation_timer, &QTimer::timeout,
+            this, [=]() { this->update(); });
+    connect(&m_delay_timer, &QTimer::timeout,
+            this, [=]() { this->m_animation_timer.start(); });
+    m_delay_timer.start();
+}
+
+void BlockClockDial::setupConnectingGradient(const QPen & pen)
+{
+    m_connecting_gradient.setCenter(getBoundsForPen(pen).center());
+    m_connecting_gradient.setAngle(m_connecting_start_angle);
+    m_connecting_gradient.setColorAt(0, m_confirmation_colors[5]);
+    m_connecting_gradient.setColorAt(0.5, m_confirmation_colors[5]);
+    m_connecting_gradient.setColorAt(0.6, m_confirmation_colors[4]);
+    m_connecting_gradient.setColorAt(0.7, m_confirmation_colors[3]);
+    m_connecting_gradient.setColorAt(1, m_background_color);
```

Comment:
Can you make this change here:

https://github.com/bitcoin-core/gui-qml/compare/6d97c0a15051caab822a4f55ad79c5bc886bdf6b..3d4a76014f1cac0da0c23c1869af531233f7f377